spec/controllers/v0/decision_review_evidences_controller_spec.rb

```diff
@@ -161,6 +161,70 @@
           expect(JSON.parse(response.body)).to eq(expected_response_body)
         end
       end
+
+      context 'and the use hexapdf feature switch is disabled' do
+        before { Flipper.disable :decision_review_use_hexapdf_for_encrypted_attachments }
+
+        it 'creates a FormAttachment, logs formatted error, and increments statsd' do
```

Comment:
```suggestion
        it 'creates a FormAttachment, logs formatted success message, and increments statsd' do
```
looks like this covers the success case based on `expect(Rails.logger).to receive(:info).with(encrypted_log_params_success)`